pkg/storage/stores/shipper/bloomshipper/client.go

```diff
@@ -75,7 +73,7 @@ type MetaSearchParams struct {
 type MetaClient interface {
 	// Returns all metas that are within MinFingerprint-MaxFingerprint fingerprint range
 	// and intersect time period from StartTimestamp to EndTimestamp.
-	GetMetas(ctx context.Context, params MetaSearchParams) ([]Meta, error)
+	GetMetas(ctx context.Context, metas []MetaRef) ([]Meta, error)
```

Comment:
Should probably be

```suggestion
	GetMetas(ctx context.Context, metas []Meta) ([][]Meta, []Fetcher, error)
```

to adhere with the chunk client interface.

Reply:
Yeah, there's some consistency there, although I think I prefer separate structures for items vs their references, as it makes it very clear what's expected of the data type (no confusion about whether it's loaded yet, etc)